.eslintrc.json

```diff
@@ -19,6 +19,11 @@
     "linebreak-style": 0,
     "global-require": 0,
     "eslint linebreak-style": [0, "error", "windows"],
+    "jsx-a11y/no-noninteractive-element-interactions": [0],
+    "jsx-a11y/click-events-have-key-events": [0],
+    "jsx-a11y/control-has-associated-label": [0],
+    "no-case-declarations": 0,
+    "react/button-has-type": [0],
```

Comment:
Turn them back on and refer to the documentation.

Reply:
Feedback implemented